### PR TITLE
refactor people.js for support tags and badge for titled writers

### PR DESCRIPTION
**Pull request by charlescrepeau-rc**

### CODE REVIEW

File: server/util/personas.js [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
  const personaA = {
     key: 'a',
-    name: 'Alice Adams',
-    face: 'face9.jpeg'
+    name: 'Emeka Eze',
```
Comment (robennals) [inline server/util/personas.js:5]: Why did you rename personA to have the same name as personE? 

I assume this was accidental?

File: client/component/people.js [hunk 2]
```diff
@@ -114,71 +113,57 @@ const AnonymousFaceStyle = StyleSheet.create({
...
     }
 })
 
+export function Byline({ type = 'small', photoType = null, clickable = true, userId, name = null, photo = null, time, subtitleLabel, subtitleParams = {}, underline = false, edited = false }) {
 
-
-
-export function Byline({type='small', photoType=null, clickable=true, userId, name=null, photo=null, time, subtitleLabel, subtitleParams={}, underline=false, edited=false}) {
-    const s = BylineStyle
+    const { tagByline } = useConfig();
```
Comment (robennals) [inline client/component/people.js:137]: Can we call this bylineTag instead of tagByline?

That follows the convention used for other config slots, where the name starts with the context in which it appears.

File: client/component/people.js [hunk 3]
```diff
@@ -19,14 +20,34 @@ export function ProfilePhoto({userId, type='large', photo=null, faint=false, che
...
-            return <AnonymousFace faint={faint} type={type} border={border} />    
+            return <AnonymousFace faint={faint} type={type} border={border} />
         }
     }
+
+}
+
+function renderLayers(type, persona) { 
+
+    const { layersByline } = useConfig();
```
Comment (robennals) [inline client/component/people.js:38]: can we call this profilePhotoLayers?

Normal convention is that the name of a config slot starts with the context in which it appears.

Plus, it seems that the layers apply to any profile photo, not just those that appear in bylines, right?

**Review by robennals — CHANGES_REQUESTED** [review]
This mostly looks good. Only really two issues:
* Config slot name should start with the context
* It looks like you accidentally changed the name of one of the test personas

File: server/util/personas.js (hunk 1 above)
Comment (charlescrepeau-rc) [reply server/util/personas.js:5]: yes im reverting all file from the PR except people.js

File: client/component/people.js (hunk 2 above)
Comment (charlescrepeau-rc) [reply client/component/people.js]: updated

File: client/component/people.js (hunk 3 above)
Comment (charlescrepeau-rc) [reply client/component/people.js]: yes you're right , naming updated
